### PR TITLE
Replaces Mediborg Acid Spray with Poison Hypospray

### LINKED ISSUE #12383: Medborg emagged acid spray is useless as it barely does damage and doesn't melt items
It barely does any damage, and doesn't melt items. It's completely useless compared to how powerful it was before the acid refactor.

### PR DESCRIPTION
**Pull request by Boredone**
Hopefully did this right.
This should replace the rather weak and mostly useless Acid Spray with a Hypospray with a variety of poisons.
Honestly probobly should switch out some of the poisons, but should be fine for now.

As a note, if anyone knows/can figure out a way to make it possible to add this chems on injection to the standard Hypo, I would be happy to change it so it works that way.

Fixes #12383

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Good stuff

**Comment by kevinz000** [comment]
Still not as strong as acid.
The person has time to scraem if you don't mutetox first, and if you dO mutetox first they have time to robust you!

**Comment by Boredone** [comment]
Actually, main advantage personally is that people can't tell too easily you are poisoning them instead of healing them, only difference is if they examine you and notice the different sprite
(Which is mostly so Mediborgs can tell which hypo to use)

File: code/modules/reagents/reagent_containers/borghydro.dm [hunk 1]
```diff
@@ -127,6 +127,10 @@ Borg Hypospray
 	if(empty)
 		usr << "<span class='warning'>It is currently empty! Allow some time for the internal syntheszier to produce more.</span>"
 
+/obj/item/weapon/reagent_containers/borghypo/hacked
+	icon_state = "borghypo_s"
+	reagent_ids = list ("facid", "mutetoxin", "cyanide", "sodium_thiopental", "heparin", "teslium")
```
Comment (TheNightingale) [inline code/modules/reagents/reagent_containers/borghydro.dm]: What do these chemicals do when injected into someone? Mutetoxin mutes them, but the others?
(And does injecting fluorosulfuric acid into someone melt their stuff?)

Comment (phil235) [inline code/modules/reagents/reagent_containers/borghydro.dm]: >(And does injecting fluorosulfuric acid into someone melt their stuff?)

No, it does tox damage.

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 2]
```diff
@@ -157,17 +157,12 @@
...
-	emag.name = "Fluacid spray"
+	emag = new /obj/item/weapon/reagent_containers/borghypo/hacked(src)
 
 	fix_modules()
 
 /obj/item/weapon/robot_module/medical/respawn_consumable(mob/living/silicon/robot/R, coeff = 1)
 	..()
-	if(R.emagged && istype(emag, /obj/item/weapon/reagent_containers/spray))
-		emag.reagents.add_reagent("facid", 2 * coeff)
-
```
Comment (phil235) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: if you remove the entirety of the proc you can remove it all.

File: code/modules/reagents/reagent_containers/borghydro.dm (hunk 1 above)
Comment (Boredone) [inline code/modules/reagents/reagent_containers/borghydro.dm]: FAcid does Brute when Injected (I think around 18 per injection if I read it right.)
Cyanide is a standard toxin that lasts awhile with a small RNG chance to stun.
Sodium Thiopental is a Traitor Chem that puts you to sleep after awhile, while doing alot of stamina damage.
Heparin makes them bleed and does some brute damage over time.
Teslium randomly shocks them, ignoring any protection.
Overall, wouldn't mind switching out some of em if you have any suggestions.

Comment (KazeEspada) [inline code/modules/reagents/reagent_containers/borghydro.dm]: Add Lexorin for that signature choking feeling.

**Comment by Buggy123** [comment]
I like this way more than the fluacid spray, but IMO shouldn't we buff acid a bit too? It's weak as fuck.

**Comment by Boredone** [comment]
Should I have switched out Heparin instead then I suppose? For now atleast I kinda like the idea of having it hold the same amount of poisons as the normal spray does.

**Comment by KorPhaeron** [comment]
Make mediborgs great again

**Pull request merged**